calendar/calendar.yaml

```diff
@@ -25,10 +25,32 @@
 # Current meetings
 ####################################################################################################
 - id: kf032
-  name: Kubeflow Community Call (US + EMEA)
-  date: 08/18/2020
+  name: Kubeflow Community Call
+  date: 04/15/2025
   time: 8:00AM-8:55AM
-  frequency: weekly
+  frequency: bi-weekly
+  video: https://zoom.us/j/99152427566?pwd=S0djc1FPcXkyNUdneHg3UFR3VTcyQT09
+  attendees:
+    - email: kubeflow-discuss@googlegroups.com
+  description:
+    - |
+      Notes & agenda: https://bit.ly/kf-meeting-notes
+
+      Join with Zoom: https://zoom.us/j/99152427566?pwd=S0djc1FPcXkyNUdneHg3UFR3VTcyQT09
+      Meeting ID: 991 5242 7566
+      Passcode: 645928
+
+      Join with Phone (USA): +1 669 900 6833 or +1 646 558 8656
+      International numbers: https://zoom.us/zoomconference?m=Os1EjlUlpb2_XUMaQ6dX1azqMK5CkfWH
+
+      Meeting Host Link: https://zoom.us/s/99152427566
+  organizer: autobot@kubeflow.org
+
+- id: kf043
```

Comment:
Shoulnd't we add time zone like https://github.com/kubeflow/community/blob/5683537c7222607d1635dae6845640f51668a1a0/calendar/calendar.yaml#L189

Reply:
@franciscojavierarceo  we should add the time zone.

Reply:
The default timezone is PST: https://github.com/kubeflow/community/blob/master/calendar/calendar_import.py#L55